src/encoded/commands/generate_annotations.py

```diff
@@ -189,7 +189,9 @@ def get_rows_from_file(file_name, row_delimiter):
     response = requests.get(file_name)
     rows = response.content.decode('utf-8').split(row_delimiter)
     header = rows[0].split('\t')
-    zipped_rows = [dict(zip(header, row.split('\t'))) for row in rows[1:]]
+    # remove the leading and ending double-quote string 
```

Comment:
Sorry I didn't explain throughly when we talked.  I don't think this is what we're doing here. 

The original tsv file
```
https://www.encodeproject.org/files/ENCFF277WZC/@@download/ENCFF277WZC.tsv
```
 has quoted lists like
```
"BNaC2, hBNaC2"
```
The \\" gets added when we create the annotations json.  The r.strip('\\"') removes a literal \\".  Not the quotes around the list.  

This is what I wanted you to check.  Where does the \\" get added as WE create the annotations json.  Trace the code from main(), but I expect the issue is in 'get_rows_from_file' function.  It could happen in the get request on line 189 or in the decode on line 190.   Please find out where the slash gets added.  Once we determine that we can decide where to fix it, or to add a comment explaining the issue.

Reply:
I'm looking into this a little more, but we shouldn't change the code without uploading the file again.  In any case please return the feature fix the original location in synonyms.  I just wanted a more clear comment OR a fix in a better location.  Not both.

Reply:
@zoldello Per convo, I thought an actual slash quote was inserted into the json file during creation.  I'll see if I can find out why I assumed that.

Reply:
no they are in the source file, I checked with @zoldello: (snippet)
`HGNC:26859	RBMXL3	"RNA binding motif protein, X-linked like 3"	Approved	CXorf55	FLJ40249	Xq23	AK097568	139804	ENSG00000175718	NM_001145346
HGNC:2686	DAZLA~withdrawn	"symbol withdrawn, see DAZL"	Symbol Withdrawn							
HGNC:26861	C2orf73	chromosome 2 open reading frame 73	Approved		FLJ40298	2p16.2	"BC031669, AK097617"	129852	ENSG00000177994	NM_001100396
HGNC:26862	HSF5	heat shock transcription factor 5	Approved		FLJ40311	17q23.2	BC033020	124535		XM_064190
`

Reply:
sometimes column 3 has quotes (this was missed in first version).  Column 6 often has "" around a list of items that is split.  Both cases the " should be stripped.